src/core/Akka.Streams.TestKit.Tests/TestPublisherSubscriberSpec.cs

```diff
@@ -15,7 +15,7 @@ namespace Akka.Streams.TestKit.Tests
 {
     public class TestPublisherSubscriberSpec : AkkaSpec
     {
-        protected readonly ActorMaterializer Materializer;
+        private ActorMaterializer Materializer { get; }
```

Comment:
For running [this](https://github.com/akkadotnet/akka.net/pull/2335/files/e92e271ee14fa32ed9d4ef6aca0335b2fabae351#diff-4549bb036193f22a3c487a399f396d02R34) stream.